x-pack/plugins/security_solution/public/common/components/stat_items/index.tsx

```diff
@@ -205,10 +213,13 @@ export const StatItemsComponent = React.memo<StatItemsProps>(
     from,
     grow,
     id,
+    showInspectButton = true,
```

Comment:
for these stat items, are there any that `showInspectButton` is true?

Reply:
I thought it would be better to keep the default value, but seems no need now... as all the stat_items are using visualization actions.